massa-execution-worker/src/module_cache.rs

```diff
@@ -0,0 +1,58 @@
+use massa_execution_exports::ExecutionError;
+use massa_sc_runtime::{GasCosts, RuntimeModule};
+use schnellru::{ByLength, LruMap};
+use tracing::log::warn;
+
+pub struct ModuleCache {
+    gas_costs: GasCosts,
+    cache: LruMap<Vec<u8>, (RuntimeModule, u64)>,
```

Comment:
I think you should add some documentation for:
* Vec<u8>: non compiled bytecode
* u64: initial gas cost?

Reply:
Done